backend/controllers/whereabouts/scheduledMoves.ts

```diff
@@ -24,6 +21,98 @@ export default ({ prisonApi }) => {
           text: values.description,
         }))
         .sort(sortTextAlphabetically),
+      courtEvents,
+    })
+
+  const index = async (req, res) => {
+    const { userDetails } = req.session
+    const { activeCaseLoadId } = userDetails
+    const date = req.query?.date || moment().format('DD/MM/YYYY')
+
+    const movementReasons = await prisonApi.getMovementReasons(res.locals)
+    const agencyDetails = await prisonApi.getAgencyDetails(res.locals, activeCaseLoadId)
+
+    const transfers = await prisonApi.getTransfers(res.locals, {
+      fromDateTime: isoDateTimeStartOfDay(date, 'DD/MM/YYYY'),
+      toDateTime: isoDateTimeEndOfDay(date, 'DD/MM/YYYY'),
+      agencyId: activeCaseLoadId,
+      courtEvents: true,
+      transferEvents: true,
+      releaseEvents: true,
+    })
+
+    const offenderNumbers = [
+      ...transfers.courtEvents.map((ce) => ce.offenderNo),
+      ...transfers.transferEvents.map((te) => te.offenderNo),
```

Comment:
How come we need to consider transferEvents and releaseEvents when only courtEvents are actually used?

Reply:
Release event are coming next.